cryptofeed/exchanges/bitmex.py

```diff
@@ -59,6 +60,45 @@ def _parse_symbol_data(cls, data: dict) -> Tuple[Dict, Dict]:
 
         return ret, info
 
+    def parse_order(self, data):
+        timestamp = self.timestamp_normalize(data['timestamp'])
+        order = {
+            'order_id': data['orderID'],
+            'timestamp': timestamp
+        }
+        if data.get('side'):
+            order['side'] = BUY if data['side'] == 'Buy' else SELL
+        if data.get('ordStatus'):
+            order['status'] = Bitmex.parse_order_status(data['ordStatus'])
+
+        # 0 should be True at `if` statement.
+        if data.get('orderQty') is not None:
+            order['qty'] = data['orderQty']
+        if data.get('cumQty') is not None:
+            order['cum_exec_qty'] = data['cumQty']
+        if data.get('leavesQty') is not None:
+            order['leaves_qty'] = data['leavesQty']
+        if data.get('price') is not None:
+            order['price'] = data['price']
+        if (data.get('avgPrice') is not None) or (data.get('avgPx') is not None):
+            order['average'] = (data.get('avgPrice') or data.get('avgPx') or 0)
```

Comment:
We have to generalize parsed order format.
for example, All exchanges have the same format of parsed orderbook.
bitmex.
https://github.com/block-crafters/cryptofeed/blob/94196ecfa85b939360adf1cafdd1737b888b8a48/cryptofeed/exchanges/bitmex.py#L108
okex
https://github.com/block-crafters/cryptofeed/blob/94196ecfa85b939360adf1cafdd1737b888b8a48/cryptofeed/exchanges/okex.py#L246-L253
ftx.
https://github.com/block-crafters/cryptofeed/blob/94196ecfa85b939360adf1cafdd1737b888b8a48/cryptofeed/exchanges/ftx.py#L299-L306
Like orderbook, order must also be in the same format.
I suggest following ccxt generalization. (They already consider about generalization very much)
https://github.com/ccxt/ccxt/blob/master/python/ccxt/bitmex.py#L1196-L1235 
In cryptofeed, _order methods of other exchanges(already implemented) is not considering generalization
For example, 
ftx - https://github.com/block-crafters/cryptofeed/blob/94196ecfa85b939360adf1cafdd1737b888b8a48/cryptofeed/exchanges/ftx.py#L364-L401
bybit - https://github.com/block-crafters/cryptofeed/blob/94196ecfa85b939360adf1cafdd1737b888b8a48/cryptofeed/exchanges/bybit.py#L345-L349
gemini - https://github.com/block-crafters/cryptofeed/blob/94196ecfa85b939360adf1cafdd1737b888b8a48/cryptofeed/exchanges/gemini.py#L121-L130 
These should be changed later.

Reply:
@boseok-bc 

I have reviewed ccxt generalization of order.
So, I tried to include 'clientOrderId', 'stopPrice', 'postOnly', 'orderType' to the parsed order dict. However, when access them I get `TypeError: 'builtin_function_or_method' object is not subscriptable` .

Reply:
The message is raised when you try to access builtin keyword.
ex) list['postOnly'] range['stopPrice']
Maybe there is this kind of a mistake in your code.